<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log File</string>
  </property>
  <widget class="QTextBrowser" name="logBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>951</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logCloseButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>340</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logUpdateButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
